### DISCUSSION TITLE
styled NumberInput component

### DISCUSSION THREAD

**Discussion by RichardP932** [Royal-Navy/design-system#3025] [Q&A] [answered]
Is it possible to remove the space between the number and the percent character? I thought of converting it to a string and back to a number but think that might not work.

![question_for_team](https://user-images.githubusercontent.com/3466443/151021017-af633fd6-6a32-4e5f-a600-dc1dd8e07079.png)

**Comment by k9dh3zij** [comment, marked as answer]
Hi @RichardP932 

I understand you already got an answer to this, but just wanted to answer here as well, for the sake of completeness. 

We strongly recommend against changing default styles, as it defeats one of the main purposes of having a Design System, which is to promote consistency. 

If there are good reasons why your team believes any design should change, please do let us know of the thinking/justification for the change request, so we can look into whether it's something we should change across the board.

Thank you.
